app/services/ApprovalService.scala

```diff
@@ -51,7 +52,13 @@ class ApprovalService @Inject() (
       err => Future.successful(Left(err)),
       t => {
         val (process, pages) = t
-        val processMetaSection = ApprovalProcessMeta(process.meta.id, process.meta.title, initialStatus, reviewType = reviewType)
+        val processMetaSection =
+          ApprovalProcessMeta(
+            process.meta.id,
+            process.meta.title,
+            initialStatus,
+            reviewType = reviewType,
+            processCode = process.meta.processCode.getOrElse(process.meta.id))
```

Comment:
can move handling the missing processCode to the Process so only need to handle missing value in one place